### PR TITLE
Add new volunteer Heri Yanto

### PR DESCRIPTION
**Pull request by HeriYantodotDev**
Hi! I've just added a new user

### CODE REVIEW

File: data/users/HeriYantodotDev.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Heri Yanto",
+    "bio": "Web Dev",
+    "image": "https://www.heriyanto.dev/profile.88f5a3d3.jpg",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": []
```
Comment (codewithvoid) [inline data/users/HeriYantodotDev.json:8]: can you add 2-3 web devlopment topics that you can help with?

e.g ReactJS, VueJS, NodeJS, Laravel, Django, etc

**Comment by codewithvoid** [comment]
Thanks for volunteering @HeriYantodotDev.

Added small comment.

File: data/users/HeriYantodotDev.json (hunk 1 above)
Comment (HeriYantodotDev) [reply data/users/HeriYantodotDev.json]: Thanks for the comment. 
done at my last commit: https://github.com/codewithvoid/AskDevs/pull/76/commits/369f5aa51c72b4a5f21649df548849bfe8efa29f
Thank you

**Comment by codewithvoid** [comment]
Perfect!

**Pull request merged**